<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="graphLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Из</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fromCurrency"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>В</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="toCurrency"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Текущий курс:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentRatio">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="dayButton">
         <property name="text">
          <string>День</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="weekButton">
         <property name="text">
          <string>Неделя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="monthButton">
         <property name="text">
          <string>Месяц</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yearButton">
         <property name="text">
          <string>Год</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setTimeButton">
         <property name="text">
          <string>Указать промежуток</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
